### PR TITLE
[ceph-csi] change default volumeBindingMode

### PR DESCRIPTION
**Pull request by vitaliy-sn**
## Description

volumeBindingMode changed from default to WaitForFirstConsumer

## Checklist
- [ ] The code is covered by unit tests.
- [ ] e2e tests passed.
- [ ] Documentation updated according to the changes.
- [x] Changes were tested in the Kubernetes cluster manually.

## Changelog entries

```changes
section: ceph-csi
type: chore
summary: volumeBindingMode changed from default to WaitForFirstConsumer
```

### CODE REVIEW

File: modules/031-ceph-csi/hooks/get_crds.go [hunk 1]
```diff
@@ -174,8 +174,8 @@ func isStorageClassChanged(input *go_hook.HookInput, scName, scReclaimPolicy str
 	for _, storageClass := range storageClasses {
 		sc := storageClass.(StorageClassFilter)
 		if sc.Name == scName {
-			// Annotation check and annotation in templates can be safely removed after release 1.43
-			_, migrationCompleted := sc.Annotations["migration-secret-name-changed"]
```
Comment (konstantin-axenov) [inline modules/031-ceph-csi/hooks/get_crds.go:178]: Can we keep previous migration?

Comment (vitaliy-sn) [reply modules/031-ceph-csi/hooks/get_crds.go:178]: In this case, it doesn't make sense. This and the previous migration removes the storageclass.

Comment (vitaliy-sn) [reply modules/031-ceph-csi/hooks/get_crds.go:178]: There is no check here which parameter has changed.

**Review by konstantin-axenov — APPROVED** [review]

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @konstantin-axenov. I've started the workflow for label 'e2e/run/yandex-cloud'...

**Pull request merged**